telemetry/metrics.go

```diff
@@ -21,6 +21,7 @@ const (
 	FormatDefault    = ""
 	FormatPrometheus = "prometheus"
 	FormatText       = "text"
+	ContentTypeText  = `text/plain; version=` + expfmt.TextVersion + `; charset=utf-8`
```

Comment:
Was this copied from cosmos-sdk?

Reply:
Yep! https://github.com/cosmos/cosmos-sdk/commit/0a801e1c038148f17053792ee05f7fb987c0f83d#diff-d096b827a787709c04b0b3abfc3e1a2d607348bf2412dd87823991700232e2e7R26